### ISSUE TITLE
Ignore files or directories

### ISSUE THREAD

**Issue by jaggy** [wyldstyle/wyldstyle#2]
We need a way to ignore certain directories to avoid bloat on utility files.

Currently, I have a layout for the site and the admin.

So it'd be a shame if we just stick everything in a single utility file. :/

`ws --ignore=resources/views/admin`

or add another config in the `wyldstyle.json` file.

```
{
  "breakpoints": { },
  "directory": [ ],
  "ignore": ["resources/views/admin"]
}
```

**Comment by jaggy** [comment]
I think it's better for it to be cli arguments so we can just run multiple instances of wyldstyle in the same project without referring to a single json file.

**Comment by jaggy** [comment]
FIX: The current CLI arguments doesn't store the values into an array. Just a single ignore file.

**Comment by jaggy** [comment]
Fixed multiple directories on #9
